onnx_tf/backend.py

```diff
@@ -254,6 +254,27 @@ def _get_handlers(cls, opset):
   def supports_device(cls, device):
     return common_supports_device(device)
 
+  @classmethod
+  def onnx_graph_to_tensorflow_ops(cls, graph_def, input_values,
+                                   opset=None, strict=True):
```

Comment:
Please provide basic description for the method, params, and return.

Reply:
Done!